pegasus/sites.v3/code.org/views/ai_landing_page/ai_carousel_curricula_middle_high.haml

```diff
@@ -21,14 +21,14 @@
       button_label: hoc_s(:call_to_action_view_lesson_plan),
     },
     {
-      overline: hoc_s(:module_grade_9_12),
-      title: hoc_s(:curriculum_name_csa_ai),
+      overline: hoc_s("ai_page.curricula.tile.computer_vision.overline"),
```

Comment:
I think we are in a weird spot with this one. There is one computer vision unit right now. There will be two at the time of launch. The card description and button text might be a little confusing when they get to the page and its just one curriculum but not sure its worth it for the ~1-2 week. @kelbyhawn what do you think?